lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -102,7 +102,7 @@ protected function _getPortableTableColumnDefinition($tableColumn)
 
         $dbType = strtolower($tableColumn['data_type']);
         if (strpos($dbType, "timestamp(") === 0) {
-            if (strpos($dbType, "WITH TIME ZONE")) {
+            if (strpos($dbType, "with time zone")) {
```

Comment:
Have a look at line 193 ;)
But besides that you reminded me that I forgot a unit test case for the mapping issue in the platform which I added now.